### PR TITLE
Update plugin for current master. Improved performance.

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@normanjaeckel I have still a problem with the current autoupdate of the projector. If you project e.g. a motion poll slide a lot of motionpollkeypadconnections comes in... Something looks wrong. I get all yield objects after each activate of the slide. Hope you can help me here...

File: openslides_votecollector/views.py [hunk 1]
```diff
@@ -422,7 +422,7 @@ def post(self, request, poll_id, keypad_id):
         # Mark keypad as in range and update battery level.
         keypad.in_range = True
         keypad.battery_level = request.POST.get('battery', -1)
-        keypad.save()
+        Keypad.objects.filter(keypad_id=keypad.id).update
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py]: ...update(in_range=True, battery_level=...)

File: openslides_votecollector/static/js/openslides_votecollector/site.js [hunk 2]
```diff
@@ -275,7 +275,7 @@ angular.module('OpenSlidesApp.openslides_votecollector.site', [
                                         // Stop pinging after 50 seconds.
```
Comment (normanjaeckel) [inline openslides_votecollector/static/js/openslides_votecollector/site.js]: Change comment

**Comment by normanjaeckel** [comment]
Do you want to merge this? The problems are not clearly solved.

File: openslides_votecollector/views.py (hunk 1 above)
Comment (emanuelschuetze) [inline openslides_votecollector/views.py]: done

File: openslides_votecollector/static/js/openslides_votecollector/site.js (hunk 2 above)
Comment (emanuelschuetze) [inline openslides_votecollector/static/js/openslides_votecollector/site.js]: done

**Comment by emanuelschuetze** [comment]
I'll wait with merging until our autoupdate performance problem is solved.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Ready for merge. Review if you like.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Can I merge it now?

File: openslides_votecollector/views.py [hunk 3]
```diff
@@ -471,7 +472,7 @@ def post(self, request, poll_id, keypad_id):
         # Update votecollector.
         vc.votes_received = request.POST.get('votes', 0)
         vc.voting_duration = request.POST.get('elapsed', 0)
-        vc.save()
+        vc.save(skip_autoupdate=True)
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py]: Proposal: Enable autoupdate and use `if collection_element.collection_string == '...vc...USE CORRET VALUE HERE`: return []` in projector.py as we do it for some other models. So the function does not have to run through the default else clause.

File: openslides_votecollector/views.py [hunk 4]
```diff
@@ -520,7 +521,7 @@ def post(self, request, poll_id, keypad_id):
         vc = VoteCollector.objects.get(id=1)
         vc.votes_received = request.POST.get('votes', 0)
         vc.voting_duration = request.POST.get('elapsed', 0)
-        vc.save()
+        vc.save(skip_autoupdate=True)
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py]: Here the same.

**Comment by normanjaeckel** [comment]
Ok.

**Comment by emanuelschuetze** [comment]
Ok, I'll rebase and merge it. Thanks!

**Pull request merged**